csmith_testing/compile_run.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# !!!README!!!
+# The script assumes that:
+#   - file ./build/runtime.o is present
+#   - `LINKER_SCRIPT` env var is set to absolute path to linker script
+#   - PATH contains path to a directory to bin directory of llvm-valida build
+#   - gcc-multilib is installed
+#
+# The script builds csmith testcases for x86 and for delendum arch.
+# The latter is built with 3 different optimization levels.
+# Then all 4 programs are run and it's checked that their output is the same.
+#
+# At the end of a csmith program a hash of the program's state is computed and printed.
+
+set -e
+
+rm -rf build/test_* || true
+rm -rf build/log_* || true
+
+src=$1
+if [[ ! -f "$src" ]]
+then
+    echo "File does not exist"
+    exit 1
+fi
+
+echo "File under test: $src"
+
+echo "Compiling fox x86"
+
+clang -Wno-everything -std=c11 -m32 -O3 -g -nostdinc -I. -Icsmith -Icsmith/X86 $src -o build/test_x86
+
+opts="-std=c11 -target delendum -I. -Icsmith -Icsmith/Delendum"
+optsw="$opts -Wno-everything"
+
+echo "Compiling for Delendum"
+
+echo "  Compiling testcases with different opt levels"
+
+clang $optsw -O3 -c $src -o build/test_O3.o
```

Comment:
Suggestion: use a for loop for different optimization level to make the code easier to read and maintain.

Reply:
Done